### ISSUE TITLE
Histogram bar colour changing

### ISSUE THREAD

**Issue by toni-sharpe** [toni-sharpe/brainstem-research#14]
### Problem

The histogram bars change colour when new bars are added to groups making it harder to keep track of which bar set relates to which chosen data point

### Solution

Either:

- fix the colours to the bars, though this could be very tricky as the bars are deliberately spread in colour to look different
- change the colours, but associate the colour with the selection, directly on the button, using the same logic as the primary mark class (see below)
- label the actual bars

Most preferable would be the option to associate colours directly with data points as fixing the colours I think will become very tricky to do while maintaining usability. Labelling the actual bars should be considered but would be ticketed as a new task if doing this work raised the position that it was a good idea. Having both could well be very good.

https://github.com/toni-sharpe/brainstem-research/blob/main/gui/src/sections/AxisSelector/AxisSelector.scss
